woocommerce/woocommerce-ios
# Remove Some Unused Code

While working on #2281, I found that these do not seem to be used anymore.

## Testing

1. Navigate to Orders 
2. Pick an order that has a refund
3. Confirm that you can still the product details
4. Navigate back to Orders
5. Pick a processing order.
6. Tap on Begin Fulfillment
7. Confirm that you can still see the product details.

## Reviewing 

Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
